paper/paper.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'OceanBioME.jl: A flexible environment for modelling the coupled interactions between ocean biogeochemistry and physics'
+tags:
+  - julia
+  - biogeochemistry
+  - climate
+  - ocean
+  - carbon
+authors:
+  - name: Jago Strong-Wright
+    orcid: 0000-0002-7174-5283
+    corresponding: true
+    affiliation: "1, 2"
+  - name: John R Taylor
+    affiliation: "1, 2"
+  - name: Si Chen
+    affiliation: "1, 2"
+  - name: Gregory LeClaire Wagner
+    orcid: 0000-0001-5317-2445
+    affiliation: 3
+  - name: Collaborators
+affiliations:
+ - name: Department of Applied Mathematics and Theoretical Physics, University of Cambridge, Cambridge, United Kingdom
+   index: 1
+ - name: Centre for Climate Repair, Cambridge, United Kingdom
+   index: 2
+ - name: Massachusetts Institute of Technology
+   index: 3
+date: 15 March 2023
+bibliography: paper.bib
+---
+
+# Summary
+
+``OceanBioME.jl`` is a flexible modelling environment written in Julia [@julia] for modelling the coupled interactions between ocean biogeochemistry, carbonate chemistry, and physics.
+OceanBioME can be used as a stand-alone box model, or integrated into ``Oceananigans.jl`` [@Oceananigans] simulations of ocean-flavoured fluid dynamics in one-, two-, or three-dimensions.
+As a result, OceanBioME and Oceananigans can be used to simulate the biogeochemical response across an enormous range of scales: from surface boundary layer turbulence at the meter scale to eddying global ocean simulations at the planetary scale, and on computational systems ranging from laptops to supercomputers.
+An example small-scale problem is shown in Figure 1.
+OceanBioME leverages Julia's multiple dispatch and effective inline capabilities to fuse its computations directly into existing Oceananigans kernels, thus maintaining Oceananigans' bespoke performance, memory- and cost-efficiency on GPUs in OceanBioME-augmented simulations.
+
+![Here we replicate the Eady problem where a background buoyancy gradient and corresponding thermal wind generate a sub-mesoscale eddy, roughly following the setup of Taylor (2016).
+To this physical setup, we added a medium complexity (9 tracers) biogeochemical model, some of which are shown above.
+On top of this, we added particles modelling the growth of sugar kelp which are free-floating and advected by the flow, and carbon dioxide exchange from the air.
+A key advantage of writing ``OceanBioME.jl`` in Julia is that it offers accessibility similar to high-level languages such as Python, with the speed of languages like C and Fortran and built-in parallelism.
+This means that models can be run significantly faster than the equivalent in other high-level languages.
+``OceanBioME.jl`` can run on GPUs, allowing the above model (1 km × 1 km × 100 m with 64 × 64 × 16 grid points) to simulate 10 days of evolution in about 30 minutes of computing time on an Nvidia P100 GPU.](eady_example.png)
```

Comment:
```suggestion
``OceanBioME.jl`` can run on GPUs, allowing the above model (1 km × 1 km × 100 m with 64 × 64 × 16 grid points) to simulate 10 days of evolution in about 30 minutes of computing time on an Nvidia P100 GPU. \label{fig1}](eady_example.png)
```

Reply:
Caption of figure 1 seems to contain context beyond description of the figure. E.g.,

> A key advantage of writing ``OceanBioME.jl`` in Julia is that it offers accessibility similar to high-level languages such as Python, with the speed of languages like C and Fortran and built-in parallelism. This means that models can be run significantly faster than the equivalent in other high-level languages."

Do we want this as part of the caption or part of the main text?

Reply:
I think this is probably adequately described in the main text already on line 39, so should probably just be cut from the caption?

Reply:
Yes, cut or shorten I'd say